### PR TITLE
Fix cairo docs

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
To fix https://github.com/gtk-rs/cairo/pull/237

cc @sdroege

### CODE REVIEW

**Comment by sdroege** [comment]
:+1: so let's get mine in, and then yours or ...?

**Comment by GuillaumeGomez** [comment]
This one has to go first so we can restart travis on yours.

**Comment by sdroege** [comment]
I see

**Comment by sdroege** [comment]
This needs updating for the versions too

File: .travis.yml [hunk 1]
```diff
@@ -14,9 +14,7 @@ script:
...
   - mkdir .cargo
   - echo 'paths = ["."]' > .cargo/config
-  - git clone -q --depth 50 -b pending https://github.com/rust-gnome/examples _examples
+  - git clone -q --depth 50 -b pending https://github.com/gtk-rs/examples _examples
   - cd _examples
   - ./build_travis.sh
-  - if [ "$GTK" = "3.18" ]; then FEAT="gtk_3_18"; fi
-  - echo $FEAT
-  - cargo build --features "$FEAT gtk/embed-lgpl-docs" --jobs 1
+  - cargo build --features "$FEATURES gtk/embed-lgpl-docs" --jobs 1
```
Comment (sdroege) [inline .travis.yml]: Why? `build_travis.sh` already does a build?

Comment (GuillaumeGomez) [reply .travis.yml]: Indeed! I removed it.

**Comment by sdroege** [comment]
Seems that it's now happy, good :)

**Comment by GuillaumeGomez** [comment]
Indeed! Merging then I restart your cairo PR.

**Pull request merged**